spec/system/bus_stops/managing_bus_stops_spec.rb

```diff
@@ -38,7 +38,9 @@
 describe 'viewing outdated' do
   let(:admin) { create :user, :admin }
   let!(:present_stop) { create :bus_stop }
-  let!(:date) { Date.today }
+  # default date for outdated is from a month ago
+  let!(:date) { Date.today - 1.month }
+  let(:picked_date) { date.change(day: 28) }
```

Comment:
I don't see that we use this anywhere.

Reply:
Do you mean [`picked_date`](https://github.com/umts/stop-project/pull/78/files/a90992220528668858c9c24164297dfdff6292ed#diff-feabba5eae22c49551b0e820f7aae3bdR82) or [`date`](https://github.com/umts/stop-project/pull/78/files/a90992220528668858c9c24164297dfdff6292ed#diff-feabba5eae22c49551b0e820f7aae3bdL42)? Both are used, but maybe they don't need to. Would you suggest that I ditch `date` and use `Date.today - 1.month` wherever it's needed?

Reply:
No, I just misread and didn't see where `picked_date` is used. Disregard.

Reply:
Okie dokie